doc/lsst.fgcmcal/tasks/lsst.fgcmcal.fgcmFitCycle.FgcmFitCycleTask.rst

```diff
@@ -0,0 +1,125 @@
+.. lsst-task-topic:: lsst.fgcmcal.fgcmFitCycle.FgcmFitCycleTask
+
+################
+FgcmFitCycleTask
+################
+
+``FgcmFitCycleTask`` runs a single ``fgcm`` fit on the star observations generated from :doc:`lsst.fgcmcal.fgcmBuildStars.FgcmBuildStarsTask` using the look-up table generated from :doc:`lsst.fgcmcal.fgcmMakeLut.FgcmMakeLutTask`.  This code is meant to be run multiple times until convergence, and the results output from one "fit cycle" are used as an input to the subsequent fit cycle.  One final cleanup run is performed to generate the tables required for input to :doc:`lsst.fgcmcal.fgcmOutputProducts.FgcmOutputProductsTask`.
+
+This is the third task in a typical ``fgcmcal`` processing chain.  The first is :doc:`lsst.fgcmcal.fgcmMakeLut.FgcmMakeLutTask`, the second is :doc:`lsst.fgcmcal.fgcmBuildStars.FgcmBuildStarsTask`, and the fourth is :doc:`lsst.fgcmcal.fgcmOutputProducts.FgcmOutputProductsTask`.
+
+``FgcmFitCycleTask`` is available as a :ref:`command-line task <pipe-tasks-command-line-tasks>`, :command:`fgcmFitCycle.py`.
+
+.. _lsst.fgcmcal.fgcmFitCycle.FgcmFitCycleTask-summary:
+
+Processing summary
+==================
+
+``FgcmFitCycleTask`` reads in the star observations and the look-up table, performs an atmosphere and instrument fit, and outputs fit parameters as well as a comprehensive list of QA plots.  If the config option ``isFinalCycle`` is set to ``True`` then additional datafiles are output that are used by :doc:`lsst.fgcmcal.fgcmOutputProducts.FgcmOutputProductsTask`.
```

Comment:
Might be nice to have more descriptions about how one decides and changes this to be the final cycle. Referring to the cookbook would help already.